CODE_OF_CONDUCT.md

```diff
@@ -43,81 +43,137 @@ For more information, refer to the full
 
 ---
 
-## Contributor Covenant Code of Conduct v2.0
+_Contributor Covenant Code of Conduct v2.0, from [commit 8001bf8](https://github.com/EthicalSource/contributor_covenant/blob/8001bf8c6e7cd2606657e2816710770d8a79b7dc/content/version/2/0/code_of_conduct.md)_
```

Comment:
Do we want this change in the CODE_OF_CONDUCT? Explicitly adding a version label, other than the upstream changes will create diffs, and the previous pull request needs to remove this particular change. Due to this, I don't see any good reasoning for making this change.

Reply:
This seems like maybe something that the patch script could add implicitly? then it wouldn't be shown as a diff in the patch file but would still help track this info.

Reply:
Much of the confusion in our CoC discussions has arisen from not having clarity on what changes in our CoC compared to the upstream are modifications we've made, and what has been modified in the upstream. Our current text is an essentially unmodified copy of the upstream at the specified commit, but the upstream "v2.0" has since been modified. If/as we start to actively patch our CoC, I would think that we continue to need an explicit indicator for where our text is coming from.

If we do start to use a script to apply such changes, this text will need to reflect that, possibly directing interested partied to a more detailed explanation for where to find the sources.

Reply:
> If we do start to use a script to apply such changes, this text will need to reflect that

This requires duplication of updating the CoC version, and I believe it might be a better way to use the update script to append the results for the versioning of it automatically.